packages/block-editor/src/components/list-view/index.js

```diff
@@ -264,6 +270,7 @@ export default forwardRef( ( props, ref ) => {
 			showAppender={ false }
 			blockSettingsMenu={ BlockSettingsDropdown }
 			rootClientId={ null }
+			onSelect={ null }
```

Comment:
Can't this new prop have a `null` default instead of having to put it in all list view instances?

Reply:
Does it need to be `null` at all, or can it just be the default of `undefined`? Or is it better to flag it as `null` so that it's explicit that it's not a required prop?

Reply:
The reason for setting it as null here is that it prevents people from using it in the `ListView` component, thus keeping the new props private.

Reply:
Oh, it's a reset to null after `{ ...props }` I see.

Reply:
Is it not a bit confusing that when using the `PrivateListView` named export the `onSelect` prop will work as expected, but when using the default export `onSelect` won't work? (Same thing for the other props that get overridden in the default export).

Should we at least document the default export making it clear that `showAppender`, `blockSettingsMenu`, `rootClientId` and `onSelect` are not meant to be passed to that component?